datagovharvester/extract/dcatus.py

```diff
@@ -0,0 +1,56 @@
+from .utils import download_catalog
+import json
+from datagovharvester.utils.s3_utilities import (
+    upload_to_S3,
+    create_s3_payload,
+)
+
+def store_record_in_s3( record, source_id, job_id, record_idx, S3_client, bucket_name ):
+    """ store the input record in a s3 bucket
+    record (dict)               :   dcatus record
+    source_id (str)             :   uuid 
+    job_id (str)                :   uuid
+    record_idx (int)            :   index position of record in records list
+    S3_client (boto3.client)    :   boto3 s3 client. 
+    bucket_name (str)           :   name of bucket to store data. 
+    """
+    try:
+        record = json.dumps(record)
+        key_name = f"{source_id}/{job_id}/{record_idx}.json"
```

Comment:
We should define this in a single place.  I see it referenced in two places. (https://github.com/GSA/datagov-harvesting-logic/pull/7/files#diff-9d01608e3885086371828eef6b60f3143ad31367f557b6019cde7b24559ed8f9R50)

Reply:
I agree. I'll change that now.

Reply:
[updated to use 1 reference](https://github.com/GSA/datagov-harvesting-logic/commit/7049e38c44ab4e0a1b4db38c915d849eeb6ffee6)